gsoc/admin.py

```diff
@@ -79,7 +80,30 @@ def return_func(self, request, obj=None, **kwargs):
             )
         return form
     return return_func
-
+def Article_change_view(self, request, object_id, *args, **kwargs):
```

Comment:
Try following PEP8 and PEP257 guidelines. For example: naming of this function can be changed to `article_change_view`.

Pro tip:
Use a linter to help you fix these kinds of issues. I, personally use PyCharm which automatically highlights these issues.

Another way to solve it:
- Install a linter like `flake8` using `pip install flake8 flake8-docstrings pep8-naming flake8-import-order`
- And run it `flake8 <filename>`

It will highlight the issues.

Reply:
Thank you! I use pycharm too but I think PEP8 sometimes can be annoying so I turned the linter off :0